package/kedro_viz/models/graph.py

```diff
@@ -201,7 +204,7 @@ def create_data_node(
         """
         return DataNode(
             id=cls._hash(full_name),
-            name=_pretty_name(full_name),
+            name=_pretty_name(_strip_namespace(full_name)),

```

Comment:
I noticed task node uses `_pretty_name(getattr(node, "short_name", node.name))` to apply the ruleset in their naming to eliminate the namespaces, I wonder that, instead of setting a new class, perhaps we can reuse that?